ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -581,26 +588,32 @@ private object PostgresQueries extends Queries {
 }
 
 private object OracleQueries extends Queries {
-  import Queries._
+  import Queries._, InitDdl._
   import Implicits._
 
   type SqlInterpol = Queries.SqlInterpolation.Unused
 
-  protected[this] override def dropTableIfExists(table: String) = sql"""BEGIN
-      EXECUTE IMMEDIATE 'DROP TABLE ' || $table;
+  protected[this] override def dropIfExists(d: Droppable) = {
+    val sqlCode = d match {
+      case _: CreateTable => -942
```

Comment:
I did not find them in any docs; at the very least, the reference for `CREATE *` does not mention which errors may be emitted.

-942 came from the stackoverflow that included it. But, as I found with materialized view, the second was found by running the test with the wrong number and looking at the matching `ORA-*` error message.

I can include copies of the error messages, if you like. Or just a link to this very comment. I would recommend that approach should this list need to be extended again, rather than diving into the docs again.